mesheryctl/internal/cli/root/registry/generate.go

```diff
@@ -199,8 +206,10 @@ func InvokeGenerationFromSheet(wg *sync.WaitGroup) error {
 				return
 			}
 
-			if model.Registrant == "artifacthub" {
-				time.Sleep(10 * time.Second)
+			if mutils.ReplaceSpacesAndConvertToLowercase(model.Registrant) == "artifacthub" {
+				time.Sleep(1 * time.Second)
```

Comment:
Why do we need this, when we have https://github.com/meshery/meshery/pull/11144/files#diff-9af4ad85714a082c37d38006b390c594453af5b29a63e3c272a42805d8bffef8R211 ?

Reply:
Removed it it's redundant .